gulpfile.js

```diff
@@ -134,26 +185,6 @@ gulp.task('test', ['clean'], function () {
     }));
 });
 
-//
-// Making this task depend on lint is a bit of a hack. The `run-tests` command is the entrypoint for the CI process,
-// and it needs to run all these tasks together. However, the "lint" and "mocha" tasks explode when used in parallel,
-// resulting in some mysterious "ShellJS: internal error TypeError: Cannot read property 'isFile' of undefined"
-// errors.
-//
-// Gulp doesn't support serial dependencies (until gulp 4.0... which is most likely never coming out)... so we have
-// to trick it by declaring 'lint' as a dependency here. See https://github.com/gulpjs/gulp/blob/master/docs/recipes/running-tasks-in-series.md
-//
-gulp.task('mocha', ['webpack', 'lint'], function() {
-    return gulp.src(['test/spec/loaders/**/*.js'], { read: false })
-        .pipe(mocha({
-          reporter: 'spec',
-          globals: {
-            expect: require('chai').expect
-          }
-        }))
-        .on('error', gutil.log);
-});
```

Comment:
These tests were all tests related to the webpack loaders.  The need for loaders was replaced with the introduction of modules, so this tests are no longer relevant.